task3.js

```diff
@@ -0,0 +1,54 @@
+// 1.Remove duplicates of the array and return an array of only unique elements
+const array = [1, 2, 3, 5, 1, 5, 9, 1, 2, 8];
+let unique=[...new Set(array)];
+console.log(unique)
+
+// 2.Find the intersection of two arrays. (An intersection would be the common elements that exist within both arrays). The elements should be unique!
+const firstArray = [2, 2, 4, 1];
+const secondArray = [1, 2, 0, 2];
+var commonValues = _.intersection(firstArray, secondArray);
+console.log(commonValues)
+
+// 3.Remove empty values from the array.
+const array1 = [0, 1, null, 2, "", 3, undefined, 3,,,,,, 4,, 4,, 5,, 6,,,,]
+var filtered = array1.filter(function (el) {
```

Comment:
Empty values are not just null. They are can be empty strings or undefined according to the array.